### PR TITLE
Milestone: Create elements and a Pop up window dynamically

### PR DESCRIPTION
**Pull request by kwambiee**
# In this project, I was able to;
- Remove portfolio markup from the homepage.
- Create elements with Javascript.
- Create a pop up window with javascript.
- Style all elements with CSS.

### CODE REVIEW

File: action.js [hunk 1]
```diff
@@ -19,23 +83,141 @@ window.onload = function meunWindow() {
...
+              </div>
+              <div class="modal-links">
+                <a href="${
+                  project.sourceLink
+                }"><span>See live <img src="./images/live.png"/></span></a>
+                <a href="${
+                  project.demoLink
+                }"><span>See Source<img src="./images/giticon.png"/></span>
+                </a>
+              </div>
```
Comment (thecodechaser) [inline action.js]: - [x] Great on the styles but for this project you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)design as much as possible. 
- Please place some gaps between your button's text and icon and make sure it matches the design on the template.

## Your design

image

### Figma design

image

File: action.js [hunk 2]
```diff
@@ -19,23 +83,141 @@ window.onload = function meunWindow() {
...
+        </div>
+        <div class="item-body">
+          <div class="item-details">
+            <span>${project.company}</span>
+            <span class="item-role">${project.role}</span>
+            <span>${project.year}</span>
+          </div>
+          <div class="item-img img${i + 1}">
+            <p>image point</p>
+          </div>
```
Comment (thecodechaser) [inline action.js:106]: - [x] On the mobile screens the project image is not showing, Please fix the issue and make sure the image is visible on all screens. 

### See the image for reference 

image

File: action.js [hunk 3]
```diff
@@ -19,23 +83,141 @@ window.onload = function meunWindow() {
...
 
-  open.addEventListener('click', () => {
-    div.style.display = 'block';
+  function addModals() {
+    let modalitem = "";
+    projectsArray.forEach((project, i) => {
+      let tags = "";
+      project.tags.forEach((tag) => {
+        tags += `<span>${tag}</span>`;
+      });
```
Comment (thecodechaser) [inline action.js]: - [x] Please fix the design for desktop screens. Place technologies, and buttons to the right side after the project description and make sure it matches the design on the template.

### Your design

image


### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @kwambiee👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# INVALID Code Review Request :stop_sign: 

Hi @kwambiee , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that is not running linter check for JavaScript.
Kindly, add linters for JS to your repo.
[Here they are](https://github.com/microverseinc/linters-config/tree/master/html-css-js);

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

**Comment by kwambiee** [comment]
@nicupop729 
Screenshot from 2022-04-08 06-50-20
My linters are fine. They all show tick and no errors. I have installed the linter file for JSON.

**Review by nicupop729 — APPROVED** [review]
<h1>Status: Approved 🥇 ✔</h1>
Hi @kwambiee,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

### Some highlights for you
* Correct githubflow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* Clean codebase ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**